### PR TITLE
Require the user to accept the ToS and privacy policy

### PR DESCRIPTION
**Pull request by Powersource**
fixes https://github.com/Edgeryders-Participio/multi-dreams/issues/207

This PR doesn't actually do the change itself, but it adds the script that i used to do the change. We'll be able to use it (a slight modification of it) if we need to ask users in the future to accept the ToS again.

This was just for already existing users. The normal setting in keycloak will make sure new users need to accept the ToS

### CODE REVIEW

File: api/scripts/requireAcceptToS.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  RequiredActionAlias,
+} = require("keycloak-admin/lib/defs/requiredActionProviderRepresentation");
+require("dotenv").config({ path: "api/.env" });
+const initKcAdminClient = require("../utils/initKcAdminClient");
+
+if (!process.env.KEYCLOAK_AUTH_SERVER) {
+  throw ".env not loaded. Call from the top dir in the repo";
+}
+
+const TERMS = RequiredActionAlias.terms_and_conditions;
```
Comment (gustavlrsn) [inline api/scripts/requireAcceptToS.js:11]: We should also add this required action in the `inviteEventMembers` and `inviteOrgMembers` resolvers

Comment (Powersource) [reply api/scripts/requireAcceptToS.js:11]: I believe that would probably be taken care of by the setting in keycloak that requires the ToS to be accepted by new users (that I've enabled). But maybe we need to test that.

Comment (gustavlrsn) [reply api/scripts/requireAcceptToS.js:11]: Tested it now, it becomes part of the "Complete signup" process as the last step. It's not listed as one of the steps on the first screen shown after clicking that link though

Comment (Powersource) [reply api/scripts/requireAcceptToS.js:11]: Ok good. That's how the ToS accepting works in keycloak by default. You accept it on the screen after registering/loggin in. It's a bit unusual but works. I think I read you could technically maybe hack in a checkbox on the registering screen but I don't know if that would be worth the effort.

Can I mark this as resolved?

**Pull request merged**